qa/suites/rgw/notifications/tasks/http/supported-distros/$

```diff
@@ -0,0 +1 @@
+qa/suites/rgw/notifications/tasks/kafka/supported-distros/$
```

Comment:
i think you shoudl link here: `../../.qa/distros/supported-random-distro$/`
not to kafka